Healthy/Classes/Modules/Dashboard/Views/FoodTagsView/Layout/UICollectionViewLayout+Extension.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+extension UICollectionViewLayout {
```

Comment:
We only add nonprivate extensions if we're willing to use it it multiple places, otherwise, you may need to make it private extension in the same file.

Reply:
Not updated yet, please let me know if we need to talk.